src/worker-helpers.js

```diff
@@ -13,6 +13,12 @@ const Cache = {
   LAST_MODULES_PATH: null
 }
 
+const cleanPath = (path) => {
```

Comment:
__*Preference*__

It might be good to drop a small docblock here to explain what this is doing to clean up the path.